src/drawing/text.rs

```diff
@@ -42,33 +55,30 @@ pub fn text_size(scale: Scale, font: &Font, text: &str) -> (i32, i32) {
 /// `scale` is augmented font scaling on both the x and y axis (in pixels).
 ///
 /// Note that this function *does not* support newlines, you must do this manually.
-pub fn draw_text_mut<'a, C>(
-    canvas: &'a mut C,
+pub fn draw_text_mut<C>(
+    canvas: &mut C,
     color: C::Pixel,
-    x: i32,
-    y: i32,
-    scale: Scale,
-    font: &'a Font<'a>,
-    text: &'a str,
+    x: u32,
+    y: u32,
```

Comment:
I thought it would make more sense make these coordinates non-negative, but a user of this library might decide to draw string starting from out of bounds of this image continuing into it. We are already doing a bounds check at 78, so it might be better to keep them as `i32`

Reply:
Yeah let's keep them at `i32` and see what happens. That would also be less API breakage.

Reply:
Agreed, I updated :+1: